test/minitest/test_minitest_mock.rb

```diff
@@ -495,4 +495,18 @@ def test_mock_with_yield
     @tc.assert_equal true, rs
   end
 
+  def test_blow_up_on_multiple_stub
+    h = {}
+    e = assert_raises StubError do
+      h.stub(:empty?, false) do
+        h.stub(:empty?, true) do
```

Comment:
Right. I think where @zenspider is trying to get is, to understand whether this is a bad enough practice that we don't even have an error for it.